src/themes/five/src/components/Checkout/index.js

```diff
@@ -483,6 +492,11 @@ const CheckoutUI = (props) => {
               {t('WARNING_INVALID_PRODUCTS', 'Some products are invalid, please check them.')}
             </WarningText>
           )}
+          {[3, 4].includes(options?.type) && !cart?.place && (
```

Comment:
please don't use this validation because is hardcoded, create a constant for this